source/iface/math-array-types.rst

```diff
@@ -12,18 +12,18 @@ SYCL provides an ``marray<typename DataT, std::size_t NumElements>``
 class template to represent a contiguous fixed-size container. This
 type allows sharing of containers between the host and its SYCL devices.
 
-The ``marray`` class is templated on its element type and number of
+The ``sycl::marray`` class is templated on its element type and number of
 elements. The number of elements parameter, ``NumElements``, is a
 positive value of the ``std::size_t`` type. The element type
-parameter, ``DataT``, must be a numeric type as it is defined
+parameter, ``sycl::DataT``, must be a numeric type as it is defined
```

Comment:
`DataT` is a template parameter and not a concrete type, so I don't think it (or any other template parameters) should be prefixed with `sycl::`.  I might have missed this in previous PRs, so can you please update recent PRs if they also need updating to fix this?

Reply:
@mkinsner https://github.com/KhronosGroup/SYCL_Reference/pull/175/commits/207afae611e4db9cf21ffe27d1cbd2041b952e04 Please check if you was thinking in this direction? I was founded only access spacial case, from Member functions removes sycl:: 
   _template <sycl::access::address_space AddressSpace, sycl::access::decorated IsDecorated>
   void load(size_t offset, sycl::multi_ptr<const DataT, AddressSpace, IsDecorated> ptr)_

Reply:
Yes, I think that handles it.  Thanks.